tests/prague/eip7692_eof_v1/eip7620_eof_create/test_subcontainer_validation.py

```diff
@@ -19,6 +21,12 @@
     code=Op.EOFCREATE[0](0, 0, 0, 0) + Op.SSTORE(slot_code_worked, value_code_worked) + Op.STOP,
     max_stack_height=4,
 )
+eofcreate_revert_code_section = Section.Code(
+    code=Op.EOFCREATE[0](0, 0, 0, 0)
+    + Op.SSTORE(slot_code_worked, value_code_worked)
+    + Op.REVERT(0, 0),
+    max_stack_height=4,
```

Comment:
```suggestion
```
We could also take this opportunity to remove this type of line from these containers.